pom.xml

```diff
@@ -149,7 +149,32 @@
         <dependencies>
             <!-- Production Deps have been moved to google-cloud-bigtable-deps-bom -->
 
-            <!-- Test Deps in alphabetical order -->
+            <dependency>
+        <groupId>com.google.api.grpc</groupId>
```

Comment:
why are these getting added?

Reply:
Owlbot adds dependencies for all modules in the parent pom. It's one of the features helping it to propose a new client library version without human intervention.

Does this affect the library build?

Reply:
This is also just the dependency management section, not the actual dependencies